specification/maps/data-plane/Common/preview/1.0/common.json

```diff
@@ -558,14 +558,15 @@
       ],
       "x-ms-enum": {
         "name": "ResponseFormat",
-        "modelAsString": true,
+        "modelAsString": false,
         "values": [
           {
             "value": "json",
             "description": "[The JavaScript Object Notation Data Interchange Format](https://tools.ietf.org/html/rfc8259)"
           }
         ]
       },
+      "x-ms-client-default": "json",
```

Comment:
This had no effect in the code, but we'll probably eliminate XML anyway, so no big changes here.

Reply:
I also wonder why we have TextFormat and ResponseFormat

Reply:
In JS SDK, I can get rid of passing "json" as the first parameter for the [method](https://github.com/alankashiwa/azure-sdk-for-js/blob/5ce0cbb1bf621233ae2bfe36be805c95f4845312/sdk/maps/maps-geolocation/samples/v1/typescript/src/geolocation.ts#L66) because of this change.

Reply:
Travis has commented on TextFormat/ResponseFormat [here](https://github.com/Azure/azure-rest-api-specs/pull/16020/files#r707618641) as well. I'll take a look.

Reply:
While I was going through Travis comment on this one, I had same feeling that since the description for our current `ResponseFormat` is stated that `Only json format is supported.` 
However in `TextFormat`, the value can be either `_json_ or _xml_`. 
So I thought on just either make the current `TextFormat` to be `ResponseFormat` for supporting both `json` and `xml` (if needed) or just make `x-ms-client-default` to json like @alankashiwa did above. 
This is also affecting Python SDK [parameter](https://github.com/alextts627/azure-sdk-for-python/blob/maps-sdk-onboard/sdk/maps/azure-maps-samples/geolocation.py#L23) too.

Reply:
Good point @alextts627 . I think at the end of the day we'll kill XML and this will be a moot point, but for now if we could at least unify the formats it would be a huge gain already.

Reply:
@alextts627 @daniel-rocha I renamed `ResponseFormat` as `JsonFormat` and `TextFormat` as `ResponseFormat` as suggested.